Utils/hdinsight-node-common/pom.xml

```diff
@@ -189,6 +189,11 @@
             <artifactId>httpclient</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>0.1.54</version>
+        </dependency>
```

Comment:
remove the version from this pom and add it to dependencysmanagement in Utils/pom.xml